src/test/java/org/terasology/simpleFarming/systems/BaseAuthorityTest.java

```diff
@@ -14,6 +15,7 @@
 import org.terasology.engine.world.block.Block;
 import org.terasology.moduletestingenvironment.ModuleTestingHelper;
 
+@Tag("MteTest")
 public abstract class BaseAuthorityTest {
```

Comment:
hmm, not sure what to think about using this on the abstract class that doesn't invoke MTE itself.

Reply:
fair point... removed :+1: